CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to Mapepire Java Client SDK
+
+Thank you for considering contributing to the Mapepire Java Client SDK! We appreciate your support and are excited to collaborate with you. This document outlines the process for contributing to the project.
+
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [How Can I Contribute?](#how-can-i-contribute)
+  - [Reporting Bugs](#reporting-bugs)
+  - [Suggesting Enhancements](#suggesting-enhancements)
+  - [Submitting Changes](#submitting-changes)
+- [Development Workflow](#development-workflow)
+  - [Getting Started](#getting-started)
+  - [Creating a Branch](#creating-a-branch)
+  - [Making Changes](#making-changes)
+  - [Committing Changes](#committing-changes)
+  - [Submitting a Pull Request](#submitting-a-pull-request)
+- [Style Guide](#style-guide)
+- [License](#license)
+- [Acknowledgements](#acknowledgements)
+
+## Code of Conduct
+
+By participating in this project, you agree to abide by our [Code of Conduct](CODE_OF_CONDUCT.md). Please read it to understand the expectations for our community.
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+If you find a bug, please report it by opening an issue on our [GitHub Issues page](https://github.com/your-username/mapepire-java/issues). Provide as much detail as possible, including steps to reproduce the issue, your environment, and any relevant logs.
+
+Before you start working on a bug, please comment on the issue to let us know that you are working on it. This helps prevent duplicate efforts.
+
+### Suggesting Enhancements
+
+We welcome new ideas! If you have a feature request or enhancement suggestion, please open an issue on our [GitHub Issues page](https://github.com/your-username/mapepire-java/issues) and describe the enhancement you'd like to see. 
+
+### Submitting Changes
+
+If you would like to contribute code to the project, follow the steps below to get started.
+
+## Development Workflow
+
+### Getting Started
+
+1. Fork the repository to your own GitHub account.
+2. Clone your forked repository to your local machine:
+
+    ```sh
+    git clone https://github.com/your-username/mapepire-java.git
+    cd mapepire-java
+    ```
+
+3. Install the necessary dependencies:
+
+    ```sh
+    ./gradlew build
+    ```
+
+### Creating a Branch
+
+Create a new branch for your changes. Use a descriptive name for the branch to reflect the work being done.
+
+## Making Changes
+Make your changes to the codebase. Ensure that your changes adhere to our [style guide](#style-guide).
+
+## Committing Changes
+Commit your changes with a descriptive commit message. Follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification for your commit messages.
+
+
+git add .
+git commit -m "feat: add new feature description"
```

Comment:
This should be wrapped in a `sh` code block and should include the `-s` option to sign commits:

```sh
git add .
git commit -s -m "feat: add new feature description"
```